src/c64/mmu.cpp

```diff
@@ -39,12 +50,12 @@ MMU::MMU(EventScheduler &scheduler, IOBank* ioBank) :
     zeroRAMBank(*this, ramBank),
     seed(3686734)
 {
-    cpuReadMap[0] = &zeroRAMBank;
+    cpuReadMap[0] = &Read<ZeroRAMBank, &MMU::zeroRAMBank>;
```

Comment:
Not sure if it helps but maybe typedefing these templates could improve readability?

Reply:
It's not a template, but template function. So it cannot be typedef'ed

Reply:
Whoops, my bad.
In c++14 it could have be done with a variable template but not a big issue...